lib/streamlit/elements/arrow.py

```diff
@@ -313,7 +314,7 @@ def dataframe(
         self,
         data: Data = None,
         width: Width = "stretch",
-        height: int | Literal["auto"] = "auto",
+        height: HeightWithoutContent | Literal["auto"] = "auto",
```

Comment:
`"stretch"` probably also needs to be added to the docstring? And I think you can directly add this to `data_editor` API as well since it's using the exact same frontend. implementation